src/translations/translations.en.js

```diff
@@ -213,6 +213,64 @@ export const translations = {
+  <p>Keep an eye on our <a href=https://developers.clever-cloud.com/changelog/">changelog</a> or check our <a href="https://developers.clever-cloud.com/api/v4/#load-balancers">v4 API documentation</a> for this.</p>`,
+  'cc-domain-management.dns.a.heading': `A records`,
+  'cc-domain-management.dns.cname.desc': () => sanitize`<p>Using a <code>CNAME</code> record is highly recommended. This automatically keeps your configuration up to date.</p>`,
+  'cc-domain-management.dns.cname.heading': `CNAME record`,
+  'cc-domain-management.dns.desc': () => sanitize`<p>To associate a domain managed by a third-party provider to your Clever Cloud application, you need to configure its DNS zone.</p>
+  <p>This may be achieved using a <code>CNAME</code> or <code>A</code> records. We recommend using a <code>CNAME</code> record, as you won't have to reconfigure it if we change our IPs.</p>`,
+  'cc-domain-management.dns.heading': `Configure your DNS`,
+  'cc-domain-management.dns.info.apex': () => sanitize`For APEX domains or subdomains with an existing DNS zone, refer to our <a href="https://developers.clever-cloud.com/doc/administrate/domain-names/">DNS & Domains documentation</a>.`,
+  'cc-domain-management.dns.info.heading': `Dedicated load balancers & specific cases`,
+  'cc-domain-management.dns.info.load-balancer': `If you are using a dedicated load balancer, refer to its configuration or contact support. Our team can also help you to order such a service.`,
+  'cc-domain-management.dns.loading-error': `Something went wrong while loading DNS information.`,
+  'cc-domain-management.form.domain.error.contains-path': ({ path }) => `Enter the "${path}" part within the "Path" input field`,
+  'cc-domain-management.form.domain.error.empty': `Enter a domain`,
+  'cc-domain-management.form.domain.error.format': `Enter a valid domain, for instance "example.com"`,
+  'cc-domain-management.form.domain.error.wildcard': () => sanitize`Enter a valid domain.<br>Wildcards "*" may only be used as subdomains like: "*.example.com"`,
+  'cc-domain-management.form.domain.help': () => sanitize`For instance: <code>example.com</code>, <code>*.example.com</code> or <code>example.cleverapps.io</code>`,
+  'cc-domain-management.form.domain.label': `Domain name`,
+  'cc-domain-management.form.info.cleverapps': () => sanitize`If you need a domain name for testing purposes, you may use <code>my-website.cleverapps.io</code>`,
+  'cc-domain-management.form.info.docs': () => sanitize`Associate a domain name to your application. For more information, refer to our documentation <a href="https://developers.clever-cloud.com/doc/administrate/domain-names/">Domain Names</a>.`,
+  'cc-domain-management.form.info.primary': `You can choose a primary domain. Links to your application from within the console lead to this domain and emails about your application only mention this domain.`,
```

Comment:
question: there are no mention of the path prefix in those texts, should we add some?